includes/widgets/progress.php

```diff
@@ -350,12 +359,17 @@ protected function render() {
 	protected function _content_template() {
 		?>
 		<#
+		let progress_percentage = 0;
+		if ( ! isNaN() ) {
```

Comment:
Did you mean `isNaN( settings.percent.size )`?

Reply:
walla... done.